재호/17413.py

```diff
@@ -0,0 +1,35 @@
+def append_reverse_word_to_stk(stk, word_stk):
+    word_stk.reverse()
+    stk += word_stk
+
+
+def solution():
+    S = input()
+    stk = []
+    word_stk = []
+    flag = False
+    for char in S:
+        if char == '<':
+            flag = True
+            if len(word_stk) > 0:
```

Comment:
```suggestion
            if word_stk:
```
word_stk이라는 리스트가 비었는지 확인하는 방법으로 위와 같이 리스트의 empty 여부를 확인하는 방법도 있습니다!

Reply:
오! 감사합니다! 👍